packages/control-input/src/ControlGroup.tsx

```diff
@@ -37,44 +38,49 @@ export const ControlGroup = ({
 }: ControlGroupProps) => {
 	const { groupId, hintId, messageId } = useControlGroupIds(id);
 	const describedByIds = [
-		hint ? hintId : null,
 		message ? messageId : null,
+		hint ? hintId : null,
 	].filter(Boolean);
-	const describedBy = describedByIds.length
+	const describedByIdString = describedByIds.length
```

Comment:
Could we keep this as `describedBy`? I believe we use this naming convention in other packages (Field, Fieldset)